<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLogin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPass">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLogin">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>50</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPass">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>100</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>65</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
